docs/src/introduction_to_catalyst/introduction_to_catalyst.md

```diff
@@ -1,7 +1,7 @@
 # [Introduction to Catalyst](@id introduction_to_catalyst)
 In this tutorial we provide an introduction to using Catalyst to specify
 chemical reaction networks, and then to solve ODE, jump, and SDE models
-generated from them. At the end we show what mathematical rate laws and
+generated from them[1]. At the end we show what mathematical rate laws and
```

Comment:
If this is supposed to be a reference then I'd just convert the list back to a footnote. I only switched it because it was giving errors when it was unreferenced.

Reply:
Ahh, was a bit unsure if that's what you were doing. I will check how the printed docs looks like, but converting back to a list makes sense.